src/ai/backend/client/cli/vfolder.py

```diff
@@ -2,17 +2,20 @@
 import json
 from pathlib import Path
 import sys
+from tqdm import tqdm
```

Comment:
Reorder the imports consistently.
(Q: What is the rule?)

Reply:
Should I reorder it in alphabetical order?